src/vast_tracker.js

```diff
@@ -485,9 +539,11 @@ export class VASTTracker extends EventEmitter {
    * Calls the tracking URLs for the given eventName and emits the event.
    *
    * @param {String} eventName - The name of the event.
+   * @param {Object} [variables={}] - An optional Object of parameters(vast macros) to be used in the tracking calls.
    * @param {Boolean} [once=false] - Boolean to define if the event has to be tracked only once.
+   *
    */
-  track(eventName, once = false) {
+  track(eventName, variables, once = false) {
```

Comment:
where is `once` used? Because I don't see the change in the PR. This is a breaking change btw. to avoid the breaking change you should put it last with a default value `{}`. Also all the new params need to be added to the documentation. We should also add there which macros we currently support.

Reply:
Also we should mention which macros we fill automatically, and which ones need to be added by the user.

Reply:
As discussed in person, i replaced the track optionals parameters with a single object. All the documentation was in work since it was WIP, but now it's done 😉